modules/apps/foundation/portal-configuration/portal-configuration-module-configuration/src/main/java/com/liferay/portal/configuration/module/configuration/internal/ConfigurationInvocationHandler.java

```diff
@@ -65,61 +72,136 @@ public Object invoke(Object proxy, Method method, Object[] args)
 		}
 	}
 
-	private Object _invokeConfigurationOverride(Method method, Object[] args)
-		throws IllegalAccessException, InvocationTargetException,
-			   NoSuchMethodException {
-
-		Class<?> clazz = _configurationOverrideInstance.getClass();
-
-		method = clazz.getMethod(method.getName(), method.getParameterTypes());
+	private Object _getValue(
+			Class<?> returnType, String key, boolean getReturnTypeDefault)
```

Comment:
@JorgeFerrer I'll defer to you and @mikakoivisto's decision on this one since I originally used 2 different methods but can see the merit of both approaches :)